src/Microsoft.ML.Api/DataViewConstructionUtils.cs

```diff
@@ -42,6 +44,275 @@ internal static class DataViewConstructionUtils
             return new StreamingDataView<TRow>(env, data, internalSchemaDefn);
         }
 
+        public static InputRow<TRow> CreateInputRow<TRow>(IHostEnvironment env, SchemaDefinition schemaDefinition = null)
+            where TRow : class
+        {
+            Contracts.AssertValue(env);
+            env.AssertValueOrNull(schemaDefinition);
+            var internalSchemaDefn = schemaDefinition == null
+                ? InternalSchemaDefinition.Create(typeof(TRow), SchemaDefinition.Direction.Read)
+                : InternalSchemaDefinition.Create(typeof(TRow), schemaDefinition);
+
+            return new InputRow<TRow>(env, internalSchemaDefn);
+        }
+
+        public static IDataView LoadPipeWithPredictor(IHostEnvironment env, Stream modelStream, IDataView view)
```

Comment:
>public [](start = 8, length = 6)

maybe `internal`? or else document it :)

Reply:
It is internal. This is an internal class.

---
In reply to: [219577553](https://github.com/dotnet/machinelearning/pull/973#discussion_r219577553) [](ancestors = 219577553)